### PR TITLE
Скрипаль А. Д. 381806-2. ЛР2

### PR DESCRIPTION
**Pull request by drwff**
### Pull Request Readiness Checklist

See details at [code review]( https://github.com/UNN-ITMM-Software/devtools-course-practice/blob/main/lab-guide/lab-1-setup.md#%D1%80%D0%B5%D1%86%D0%B5%D0%BD%D0%B7%D0%B8%D1%8F-%D0%BA%D0%BE%D0%B4%D0%B0)

- [ ] Code is well documented and more information can be seen on the [website](https://devtools-course-practice-2021.readthedocs.io/en/latest/modules.html) (recommended)
- [ ] Make sure that the testing has passed successfully on [GitHub Actions](https://github.com/UNN-ITMM-Software/devtools-course-practice/actions)
- [ ] Get two approve from your fellow students
- [ ] Take a review from one of the administrators

### CODE REVIEW

**Comment by drwff** [comment]
@mercy131 @Sergo567

File: modules/intersection_line/src/intersection_line.cpp [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    return std::max(a, c) <= std::min(b, d);
+}
+
+int Line::area(std::pair<int, int> a, std::pair<int, int> b,
+    std::pair<int, int> c) {
+    return (b.first - a.first) * (c.second - a.second) -
+        (b.second - a.second) * (c.first - a.first);
+}
+
+Line::Line(std::pair<int, int> _a, std::pair<int, int> _b) {
```
Comment (mercy131) [inline modules/intersection_line/src/intersection_line.cpp:18]: Рекомендую организовать присваивание через список инициализации.

File: modules/intersection_line/include/intersection_line.h [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// Copyright 2021 Skripal Andrey
+#include <utility>
```
Comment (Sergo567) [inline modules/intersection_line/include/intersection_line.h:2]: Необходимо отделить пробелом копирайт и библиотеки.

File: modules/intersection_line/src/intersection_line.cpp [hunk 3]
```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 Skripal Andrey
+#include "include/intersection_line.h"
```
Comment (Sergo567) [inline modules/intersection_line/src/intersection_line.cpp:2]: Здесь аналогично.

**Review by Sergo567 — APPROVED** [review]

**Review by mercy131 — APPROVED** [review]
Теперь хорошо.

**Review by OnlyDeniko — APPROVED** [review]
👍

**Pull request merged**